core/src/main/java/com/graphhopper/util/EngineWarmUp.java

```diff
@@ -59,8 +59,8 @@ private static void warmUpCHSubNetwork(GraphHopper graphHopper, int iterations)
         Random rand = new Random(0);
 
         for (int i = 0; i < iterations; i++) {
-            int startNode = rand.nextInt(graphHopper.getMaxVisitedNodes() + 1);
-            int endNode = rand.nextInt(graphHopper.getMaxVisitedNodes() + 1);
+            int startNode = rand.nextInt(graphHopper.getRoutingConfig().getMaxVisitedNodes() + 1);
```

Comment:
Is this file even needed still?

Reply:
See https://github.com/graphhopper/graphhopper/issues/194

It is currently not used, but we or a Java user could benefit of it when using MMAP. For now I would indeed remove it and instead suggest that the user warms up JVM and caches via some random routes.